face_recognition/main.js

```diff
@@ -310,7 +310,7 @@ async function main() {
       // UI shows model loading progress
       await ui.showProgressComponent('current', 'pending', 'pending');
       console.log('- Loading weights... ');
-      const contextOptions = {deviceType};
+      const contextOptions = {'devicePreference': deviceType};
```

Comment:
Yes, since there's [a Chromium issue - Cannot test Web Neural Network API (WebNN) by specified device type](https://bugs.chromium.org/p/chromium/issues/detail?id=1447973) , here should use `devicePreference` option which is align with `MLContextOptions` of **Model Loader API** Spec to test GPU device.